ProcessLib/HydroMechanics/HydroMechanicsFEM-impl.h

```diff
@@ -152,6 +152,10 @@ void HydroMechanicsLocalAssembler<ShapeFunctionDisplacement,
         ShapeMatricesTypePressure::NodalMatrixType::Zero(pressure_size,
                                                          pressure_size);
 
+    typename ShapeMatricesTypePressure::NodalMatrixType higher_order_p =
```

Comment:
What do you mean by higher order?

Reply:
changed the name to add_p_derivative (additional_p_derivative) which suits it better.
origins in the p_fr derivative of rho_fr

Reply:
`drho_fr_dp_fr` ?

Reply:
Yes that would work too, but it probably will be extended with extra terms, when pressure and stress dependent viscosity gets implemented.